src/main/features/core/desktopSettings.js

```diff
@@ -1,6 +1,12 @@
 import { BrowserWindow } from 'electron';
 import path from 'path';
 
+var AutoLaunch = require('auto-launch');
```

Comment:
We use the standard ES6 syntax to import dependencies.

Change this to `import AutoLaunch from 'auto-launch';` please